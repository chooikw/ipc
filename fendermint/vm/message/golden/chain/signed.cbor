a1665369676e6564828a1b63672d51f7e8cf545831038a19e0d8fd53dc3ff29366ff01f3f442c74cbde2e001a7f1df1357599546d9f4009acebd713a4f4f9281ff64e1fb683c5502ea0bffcede50900080951fd940a7ba006afe488b1ba43d6d9e1d9734c4510025206a2b3bc590e292969ed36b3decdb1b2e47e9396e8ed63a510072e3450bbe5911d995e3f5be474f0af851001e23845f6cf7d101d17b7cbba4264ffc1b367de5ae680a402546e31aeb4b6d784402625aae