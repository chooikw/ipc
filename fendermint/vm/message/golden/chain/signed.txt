Signed(SignedMessage { message: Message { version: 7162743562430893908, from: Address("f25if77tw6kciabaevd7mubj52abvp4selcapom3q"), to: Address("f3rim6bwh5kpod74utm37qd47uilduzppc4aa2p4o7cnlvtfkg3h2abgwoxvytut2pska76zhb7nudyhaj2k3a"), sequence: 11834735921668240580, value: TokenAmount(49349742704242435138.209231540301196507), method_num: 3926546987369512997, params: RawBytes { e31aeb4b6d78 }, gas_limit: 3334890481968404026, gas_fee_cap: TokenAmount(152712043323050085562.853077790166551288), gas_premium: TokenAmount(40061255102063288776.007351983183122428) }, signature: Signature { sig_type: BLS, bytes: [98, 90, 174] } })